crates/wascc-host/src/middleware/runner.rs

```diff
@@ -36,7 +36,6 @@ pub(crate) async fn invoke_actor(
         error!("Middleware pre-invoke failure: {}", e);
         return Err(e);
     } else {
-        println!("invoking");
         match target.send(inv.clone()).await {
             Ok(ir) => {
                 println!("done");
```

Comment:
This seems like a temporary `println` but it should be converted to a `trace!`

Reply:
Good catch, I thought i'd gotten rid of all the temporary `println!`